package.json

```diff
@@ -66,8 +66,6 @@
     "typescript": "^3.8.3"
   },
   "husky": {
-    "hooks": {
-      "pre-commit": "yarn lint && pretty-quick --staged"
-    }
+    "hooks": {}
```

Comment:
Why remote the `pre-commit`?

Reply:
perttier keeps failing in my local environment, it appeared to be a quote vs double quote problem. An example output of `yarn lint` is:

```text
15:29  error  Replace `'casbin'` with `"casbin"`
```

There are dozens of these errors that prevent me from committing. I could not work it around quickly so I decided to remove the hook and post the PR for review. If a maintainer would kindly revert this line it would be very appreciated.

BTW, I mentioned the cause in my original post:
```text
the husky commit hook is deleted because the prettier keeps failing in my dev environment and I couldn't commit with this hook. Please can a maintainer revert this?
```